web/main/migrations/0003_alternate_legal_docs.py

```diff
@@ -12,101 +12,212 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('main', '0002_auto_20201021_1444'),
+        ("main", "0002_auto_20201021_1444"),
     ]
 
     operations = [
         migrations.CreateModel(
-            name='LegalDocumentSource',
+            name="LegalDocumentSource",
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Having Black rewrite all the Django migration files also strikes me as kind of weird, and IMO makes the migration files less readable. Maybe we should skip that directory?

Reply:
Skipping the whole migration directory makes sense to me!